admin/admin.go

```diff
@@ -43,33 +42,33 @@ import (
 	"runtime/pprof"
 	"strings"
 
+	"github.com/spf13/viper"
+
 	"github.com/rudderlabs/rudder-server/config"
 	"github.com/rudderlabs/rudder-server/services/db"
 	"github.com/rudderlabs/rudder-server/utils/logger"
 	"github.com/rudderlabs/rudder-server/utils/misc"
-
-	"github.com/spf13/viper"
 )
 
 // PackageStatusHandler to be implemented by the package objects that are registered as status handlers
-// output of Status() is expected to be json encodeable by default
+// output of Status() is expected to be json encodable by default
 type PackageStatusHandler interface {
 	Status() interface{}
 }
 
 // RegisterAdminHandler is used by other packages to
 // expose admin functions over the unix socket based rpc interface
 func RegisterAdminHandler(name string, handler interface{}) {
-	instance.rpcServer.RegisterName(name, handler)
+	_ = instance.rpcServer.RegisterName(name, handler) // @TODO fix ignored error
```

Comment:
Is the `@` prefix needed? On a side note, this is going to be one endless todo list!

Reply:
Should work either way in most editors I guess. It's just a comment anyway.

Reply:
In regards to the endless list, once I'm more confident with the codebase I'll try something. Ignoring a failure while registering a handler seems like it could be serious.